src/celengine/observer.h

```diff
@@ -305,6 +314,13 @@ class Observer
     Eigen::Vector3d 	targetVelocity{ 0.0, 0.0, 0.0 };
     Eigen::Vector3d 	initialVelocity{ 0.0, 0.0, 0.0 };
     double          	beginAccelTime{ 0.0 };
+    double              defaultGoToTime{ 5.0 };
```

Comment:
misindent

Reply:
hmm is there? they look aligned to me.

Reply:
probably you have a TAB instead of spaces

Reply:
ok apparently the other lines used some tabs instead. I've replaced all the tabs in this file with spaces